packages/components/src/Menu/Menu.css

```diff
@@ -80,8 +80,23 @@
   padding: var(--space-small);
 }
 
+:global(.jobber-retheme) .sectionHeader {
+  display: flex;
+  align-items: center;
+  min-height: var(--space-largest);
+  padding: var(--space-small) var(--space-base);
+}
+
+:global(.jobber-retheme) .sectionHeader h6 {
+  /* can move into the Typography component post re-theme */
+  font-size: var(--typography--fontSize-base);
+  font-weight: 600;
+  text-transform: none;
+}
+
 .action {
   display: flex;
+  gap: var(--space-base);
```

Comment:
removing the margin from Icon on the [old L100](https://github.com/GetJobber/atlantis/pull/1695/files#diff-487f2c6560890bbd027ea90576abb048193205478eecc1019326e553274973efL100) and converting to `gap` here